org.eclipse.lsp4j.websocket.jakarta/src/test/java/org/eclipse/lsp4j/websocket/jakarta/test/MockSession.java

```diff
@@ -1,5 +1,5 @@
 /******************************************************************************
- * Copyright (c) 2019 TypeFox and others.
+ * Copyright (c) 2019-2021 TypeFox and others.
```

Comment:
the ususal eclipse format is FirstYear, LastYear so 2019, 2021 in your case
see https://www.eclipse.org/projects/handbook/#ip-copyright-headers

Reply:
argh, will change.
Sorry, I did not care enough. Jonah provided the correct syntax in his comment already.